herbert_core/utilities/classes/@object_lookup/object_lookup.m

```diff
@@ -31,61 +45,51 @@
     %   (2) The pdf is multi-dimensional, or there is no object method called pdf_table;
     %   (3) Indexed evaluation of other methods or functions may be needed.
     %
-    % object_lookup Methods:
-    %
-    % The primary public methods are:
-    %   object_lookup   - constructor
-    %
-    %   object_array    - retrieve a given object array from the set of object arrays
-    %
-    %   object_elements - retrieve one or more elements from a given object array in the set
-    %
-    %   func_eval_ind   - evaluate a method or function for indexed occurences in the object_lookup
-    %
-    %   rand_ind        - generate random points for indexed occurences in object_lookup
-    %
     % See also pdf_table_lookup
     
     properties (Access=private)
-        % Class version number
-        class_version_ = 1;
+        % Object array of unique instance of objects in the input array or cell array
+        % (Column vector)
+        object_store_ = zeros(0,1)
```

Comment:
Don't need to initialise to 0d object generally. MATLAB defaults to `[]`

Reply:
I explicitly want the size [0,1] rather than [0,0]